packages/feature-detector/package.json

```diff
@@ -27,12 +27,16 @@
     "ts-morph": "17.0.1"
   },
   "devDependencies": {
+    "axios": "0.27.2",
     "jest-text-transformer": "1.0.4",
     "next": "13",
     "react": "18",
     "react-dom": "18",
     "tsup": "6"
   },
+  "peerDependencies": {
```

Comment:
not sure about that: is any other package using esbuild? why not add as devDependency instead?

Reply:
Discussed online: tsup is using it. Since tsup is at top level, moving esbuild there as well.